src/mitsubishi2mqtt/mitsubishi2mqtt.ino

```diff
@@ -1408,7 +1408,16 @@ void mqttCallback(char* topic, byte* payload, unsigned int length) {
     hp.update();
   }
   else if (strcmp(topic, ha_temp_set_topic.c_str()) == 0) {
+    //add to fix HP turn off after change temperature
+    heatpumpSettings currentSettings = hp.getSettings();
+    delay(10);
```

Comment:
Was there a reason for this naked 10 second delay? Seems kind of drastic!

Reply:
Syntax
delay(ms)
ms: the number of milliseconds to pause

Reply:
Doh. I think he was going for 10 seconds though